deploy-image/Dockerfile

```diff
@@ -9,29 +9,29 @@ RUN apk add --no-cache gcc musl-dev python3-dev libffi-dev openssl-dev
 
 # https://stackoverflow.com/a/57485724
 RUN apk add --update --no-cache py3-numpy py3-scipy py3-pillow
-ENV PYTHONPATH "$PYTHONPATH:/usr/lib/python3.7/site-packages"
+ENV PYTHONPATH "$PYTHONPATH:/usr/lib/python3.10/site-packages"
 
 # golang
 RUN apk add --no-cache go
 
 # aea installation
 RUN pip install --upgrade pip
-RUN pip install --upgrade --force-reinstall open-aea[all]==1.16.0
+RUN pip install --upgrade --force-reinstall open-aea[all]==1.16.0 open-aea-cli-ipfs==1.16.0
 
 # directories and aea cli config
-COPY /.aea /home/.aea
```

Comment:
not needed?

Reply:
Yeah, it was an empty config anyway. Now we initialize the repo while building the docker image